buildnode.repo

```diff
@@ -1,11 +1,5 @@
-[buildnode]
-baseurl = https://repo.almalinux.org/build_system/9/$basearch/
-enabled = 0
-gpgcheck = 0
-name = AlmaLinux - 9 - BuildNode
-
 [buildsystem]
-baseurl = https://build.almalinux.org/pulp/content/copr/eabdullin1-build_system-almalinux-9-$basearch-dr/
-enabled = 0
+baseurl = https://repo.almalinux.org/build_system/9/$basearch/
```

Comment:
9 -> $releasever, this will make system more distro-independent

Reply:
Right! Fixed in the second commit. Thanks.